docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -1,332 +1,410 @@
+   Debugging Auto-Device Plugin <openvino_docs_IE_DG_supported_plugins_AUTO_debugging>
 
-> **NOTE**: The OpenVINO Runtime lets you use "GPU" as an alias for "GPU.0" in function calls. 
-
-The Auto-device plugin supports query device optimization capabilities in metric.
-
-| Parameter name                 | Parameter values         |
-| :---                           | :---                     |
-| "OPTIMIZATION_CAPABILITIES"    | Auto-Device capabilities |
-
-### Enumerating Devices and Selection Logic
-
-The OpenVINO Runtime API now features a dedicated methods to enumerate devices and their capabilities. 
-See [Hello Query Device C++ Sample](../../samples/cpp/hello_query_device/README.md).
-This is the example output from the sample (truncated to device names only):
-
-```sh
-./hello_query_device
-Available devices: 
-    Device: CPU
-...
-    Device: GPU.0
-...
-    Device: GPU.1
-```
-
-### Default Auto-Device Selection Logic
-
-With the 2021.4 release, the Auto-Device selects the most suitable device using the following default logic:
-
-1. Check if dGPU (discrete), iGPU (integrated) and CPU devices are available
-2. Get the precision of the input model, such as FP32
-3. According to the priority of dGPU, iGPU, and CPU (in this order), if the device supports the precision of the input network, select it as the most suitable device
-
-For example, CPU, dGPU and iGPU can support the following precision and optimization capabilities:
-
-| Device   | OPTIMIZATION_CAPABILITIES       |
-| :---     | :---                            |
-| CPU      | WINOGRAD FP32 FP16 INT8 BIN     |
-| dGPU     | FP32 BIN BATCHED_BLOB FP16 INT8 |
-| iGPU     | FP32 BIN BATCHED_BLOB FP16 INT8 |
-
-* When the application uses the Auto-device to run FP16 IR on a system with CPU, dGPU and iGPU, Auto-device will offload this workload to dGPU.
-* When the application uses the Auto-device to run FP16 IR on a system with CPU and iGPU, Auto-device will offload this workload to iGPU.
-* When the application uses the Auto-device to run WINOGRAD-enabled IR on a system with CPU, dGPU and iGPU, Auto-device will offload this workload to CPU.
+@endsphinxdirective
 
-In cases when loading the network to dGPU or iGPU fails, CPU is the fall-back choice.
+The Auto-Device plugin, or AUTO, is a virtual device which automatically selects the processing unit to use for inference with OpenVINO™. It chooses from a list of available devices defined by the user and aims at finding the most suitable hardware for the given model. The best device is chosen using the following logic: 
 
-According to the Auto-device selection logic from the previous section, tell the OpenVINO Runtime 
-to use the most suitable device from available devices as follows:
+1. Check which supported devices are available. 
+2. Check the precision of the input model (for detailed information on precisions read more on the [OPTIMIZATION_CAPABILITIES metric](../IE_PLUGIN_DG/Plugin.md)) 
```

Comment:
the link is incorrect. Can we just use `ov::device::capabilities` and doxygen automatically creates a link?